cypress/integration/fetchAds.js

```diff
@@ -36,7 +36,9 @@ describe("Extract ads", () => {
             cy.get("#onetrust-banner-sdk").within(() => {
               cy.get('button[id="onetrust-accept-btn-handler"]').click()
             })
-            cy.get("#tatm-adHpmotional")
+          }
+
+          cy.get("#tatm-adHpmotional")
```

Comment:
- ohh noo! we are missing the E in `#tatm-adHpmotiona`
```suggestion
          cy.get("#tatm-adHpEmotional")
```

Reply:
thank you